Procfile

```diff
@@ -1 +1 @@
-web: node app.js
```

Comment:
One bit of feedback for you here: I'd recommend you give some positive
feedback in situations like this, especially when others are volunteering
their time to work on something of yours.

I just spent half my day working on this and you're nitpicking over the way
the menu icon was built, for example. I wouldn't do that. How about, "Hey,
you added a nav! Awesome!" People are not going to want to work with you if
you don't acknowledge their work and they don't feel appreciated.

On Tue, Apr 12, 2016 at 3:35 PM Joe Kent <notifications@github.com> wrote:

> In Procfile
> <https://github.com/deadlybutter/bernie_economics/pull/7#discussion_r59439180>
> :
>
> > @@ -1 +1 @@
> > -web: node app.js
>
> This will break the Heroku deployment, if you want to change it for
> yourself locally I can add support for a custom environment variable file.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/deadlybutter/bernie_economics/pull/7/files/f1df3aeb6f96d71b51017abf63c4620a37ac1ddb#r59439180>
>